Mathlib/Algebra/BigOperators/Basic.lean

```diff
@@ -2256,6 +2293,52 @@ theorem prod_subtype_mul_prod_subtype {α β : Type*} [Fintype α] [CommMonoid 
 #align fintype.prod_subtype_mul_prod_subtype Fintype.prod_subtype_mul_prod_subtype
 #align fintype.sum_subtype_add_sum_subtype Fintype.sum_subtype_add_sum_subtype
 
+@[to_additive] lemma prod_subset {s : Finset ι} {f : ι → α} (h : ∀ i, f i ≠ 1 → i ∈ s) :
```

Comment:
Any reason not to match `x ∉ s₁ → f x = 1` from `Finset.prod_subset`?

Reply:
It's much more useful this way, I find, as it can be interpreted as saying that the support of `f` is a subset of `s`. `Finset.prod_subset` can't really afford to be stated like that as it instead wants to emphasize `s₁ \ s₂`.